vademecum.md

```diff
@@ -52,14 +52,14 @@ Firefox offre diverse versioni, oltre alla stabile, la _Dev Edition_, rivolta ag
 La realtà virtuale oggi permette di sperimentare nuove possibilità, anche unita alla realtà aumentata.
 Queste tecnologie richiedono costosi headset ma, **grazie a Firefox**, è possibile farlo direttamente dal Web gratuitamente con la libreria JavaScript A-Frame.
 
-##### Firefox Screenshot 
- 
-Firefox Screenshot è una nuova funzione che permette di catturare, scaricare, raccogliere e condividere immagini ("screenshot") dei siti web. 
- 
-##### Pocket 
- 
+##### Firefox Screenshot
+
+Firefox Screenshot è una nuova funzione che permette di catturare, scaricare, raccogliere e condividere immagini ("screenshot") dei siti web.
+
+##### Pocket
+
 Pocket è un servizio integrato in Firefox che permette di salvare, in un unico posto, con un semplice clic, tutte le pagine di proprio interesse per poterle leggere, condividere o rimuovere successivamente.
- 
+
```

Comment:
Non ho capito cosa è cambiato... 🤔

Reply:
ha tolto gli spazi vuoti a fine riga

Reply:
ah, capito... vabbè, il vademecum.md ha valenza solo per il testo, più che per la formattazione (esclusione fatta per corsivo e grassetto)

Reply:
si deve essere stato l'editor che li ha rimossi in automatico

Reply:
Va bene :)

Reply:
Si è atom che lo fa in automatico ho provato a toglierlo ma non ci sono riuscito

Reply:
Va bene, non ti preoccupare, cercavo di capire se avessi modificato o meno qualcosa :) Don't worry :)